backend/fms_core/viewsets/experiment_run.py

```diff
@@ -41,4 +42,22 @@ def get_renderer_context(self):
     @action(detail=False, methods=["get"])
     def list_export(self, _request):
         serializer = self.serializer_export_class(self.filter_queryset(self.get_queryset()), many=True)
-        return Response(serializer.data)
\ No newline at end of file
+        return Response(serializer.data)
+
+    @action(detail=True, methods=["get"])   # TODO should this be post? update?
```

Comment:
In any case it would be patch (because we are partially updating an entry). I think we mostly use PATCH throughout FreezeMan. Refer to `set_release_status` in the dataset viewset for an example.